test/v5/testFeeHandler.js

```diff
@@ -0,0 +1,54 @@
+const Web3 = require('web3');
+const web3 = new Web3(new Web3.providers.HttpProvider('http://localhost:8545'));
+const Helper = require("../v4/helper.js");
+const BN = web3.utils.BN;
+
+const MockDAO = artifacts.require("MockDAO.sol");
+const FeeHandler = artifacts.require("FeeHandler.sol");
+const Token = artifacts.require("Token.sol");
+const BITS_PER_PARAM = 64;
+
+let kyberNetwork;
+let kyberNetworkProxy;
+let user;
+let mockDAO;
+let knc;
+let feeHandler;
+let rewardInBPS;
+let rebateInBPS;
+let epoch;
+let expiryBlockNumber;
+
+
+contract('FeeHandler', function(accounts) {
+    before("Setting global variables", async() => {
+        kyberNetwork = accounts[0];
+        user = accounts[0];
+        kyberNetworkProxy = accounts[1];
+        rewardInBPS = new BN(3000);
+        rebateInBPS = new BN(5000);
+        epoch = new BN(0);
```

Comment:
epoch and expiry block will probably update as test proceeds.
so before each could update it to current block  + x
or something like this.

Reply:
noted, added an empty beforeEach but have not used it yet... I will implement it when i need to use it otherwise for now, I will keep my test simple.

Reply:
sure